WebHostLib/templates/ootTracker.html

```diff
@@ -1,180 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why is this getting deleted in this PR?

Reply:
Because it snuck back into main somehow. The real tracker was renamed `tracker__OcarinaOfTime.html` so this isn't evne used anymore.

Reply:
I suppose it could have been in another PR.